components/select/select-top-control.component.ts

```diff
@@ -89,7 +92,18 @@ import { NzSelectItemInterface, NzSelectModeType, NzSelectTopControlItemType } f
     </ng-container>
     <nz-select-placeholder *ngIf="isShowPlaceholder" [placeholder]="placeHolder"></nz-select-placeholder>
   `,
-  host: { class: 'ant-select-selector' }
+  host: { class: 'ant-select-selector' },
+  imports: [
+    NgSwitch,
+    NzSelectSearchComponent,
+    NgSwitchCase,
+    NzSelectItemComponent,
+    NgIf,
+    NgSwitchDefault,
+    NgForOf,
```

Comment:
suggestion: ngForOr can be simplified with only ngFor

Reply:
Yeah you are right, made the change.